cpplib/Tests/GeometryTest.cpp

```
@@ -640,355 +642,496 @@ TEST(SymmTest, ValidityChecks) {
 
-	vd.calculateFaces<int>(bondlist);
-	cells = vd.extractCells();
+	cpplib::geometry::SpatialGrid<FloatingPointType> space;
+	space.build(buildresult.atoms.points, cell, 6);
+	auto bonds = space.get_bonds();
 
-	for (const auto& cell : cells) {
-		validateVoronoiCell(cell);
-	}
-}
+	voronoi::VoronoiDiagram vd(buildresult.atoms.points,bonds,cell.fracToCart(), bools);
```

Clang (14.0.6)

[warning] variable name 'vd' is too short, expected at least 3 characters

(readability-identifier-length)